rmw_zenoh_cpp/src/detail/rmw_context_impl_s.cpp

```diff
@@ -108,9 +108,17 @@ class rmw_context_impl_s::Data final : public std::enable_shared_from_this<Data>
           RMW_ZENOH_LOG_WARN_NAMED(
             "rmw_zenoh_cpp",
             "Unable to connect to a Zenoh router. "
-            "Have you started a router with `ros2 run rmw_zenoh_cpp rmw_zenohd`?");
+            "Have you started a router with `ros2 run rmw_zenoh_cpp rmw_zenohd`? "
+            "You may also increase the number of attempts to check for a router "
```

Comment:
I don't think this hint adds much. Playing with `ZENOH_ROUTER_CHECK_ATTEMPTS` is only useful if all checks are exhausted. And thus this hint would be better placed at the log below.

Reply:
Makes sense https://github.com/ros2/rmw_zenoh/pull/440/commits/dd555139cb4c0f2bf5044757cd3e7f6b0a2490fc